app/controllers/articles_controller.rb

```diff
@@ -4,7 +4,10 @@ class ArticlesController < ApplicationController
     before_action :check_for_cancel, :only => [:create, :update]
 
     def index
-        @articles = Article.paginate(page: params[:page]).limit(10).order('created_at DESC')
```

Comment:
angka limit `10` ini sebaiknya dijadikan constant dna ditaruh di atas, in case suatu saat akan di ganti, daripada dihardcode, karena gw lihat dipakai di lebih dr 1 tempat

Reply:
Okeyy notedd :D